### DISCUSSION TITLE
SCRL_MOと通常のキー入力の併存設定

### DISCUSSION THREAD

**Discussion by yataroon** [Yowkees/keyball#256] [Q&A] [answered]
お世話になります。一点質問させてください。

|キーコード| Remap上での表記 | 説明 |
| ------------- | ------------- |------------- |
SCRL_MO | 0x5DAC | キーを押している間、スクロールモードになります

特定のキーに対して、「HOLD時は上記の特殊キー、TAP時は通常のキー入力」になるような設定を
REMAPで同一レイヤー・同一キーに設定することはできますでしょうか？

お手数をおかけいたしますが、ご教示いただけると幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご質問ありがとうございます。

QMK_Firmwareの仕様に依存しまして、HOLDに設定できるキーには制限があります。
そのため、特殊キーはHOLDに配置できません。

レイヤー３に切り替わっている間に、トラックボールはスクロールモードになりますので、Layer(3)にて代用していただければと思います。

**Reply by yataroon** [reply]
ご回答ありがとうございます。
承知しました。理由も詳細に教えていただき、ありがとうございます。

最初、レイヤー３でスクロールしていたのですが、SCRL_MOの方がスクロール可能になるまでの時間が速く、スムーズに操作できるため、今はレイヤー0にSCRL_MOを置いています。

慣れの問題もあると思いますので、レイヤー３でのスクロールと場面で使い分けながら、設定を模索していきたいと思います。（そのあたりの楽しさも含めて、）素晴らしいデバイスをありがとうございます！
